content/departments/support/process/p4-enablement.md

```diff
@@ -201,19 +201,16 @@ Sub-repo permissions are still experimental and below are the steps required to
 Add this configuration to your external service configuration in dev private
 
 ```yaml
-"PERFORCE": [
+'PERFORCE':
+  [
     {
-      "p4.port": "perforce.sgdev.org:1666",
-      "p4.user": "admin",
-      "p4.passwd": "REDACTED",
-      "depots": {
-        "//test-perms/"
-      ],
-      "repositoryPathPattern": "perforce/{depot}",
-      "authorization": {
-        "subRepoPermissions": true
-      }
-    }
+      'p4.port': 'perforce.sgdev.org:1666',
```

Comment:
Looks like prettier ran automatically on this after I pushed.. I assume it's still valid?

Reply:
Oops, it was because I annotated it with `yaml` instead of `json`